src/App.jsx

```diff
@@ -1,36 +1,46 @@
+
 import {
+  useParams,
+  Switch,
   Link,
   Redirect,
   BrowserRouter as Router,
   Route,
 } from 'react-router-dom';
 
+
 import styles from './App.css';
 
 function RGB() {
+  const { r, g, b } = useParams();
+  
   // 🚨 Get values for r, g, and b from params
   // 🚨 Uncomment next lines after importing
-  // return (
-  // <div
-  //   className={styles.fill}
-  //   style={{ background: `rgb(${r}, ${g}, ${b})` }}
-  // >
-  //   <p>{`rgb(${r},${g},${b})`}</p>
-  // </div>
-  // );
+  return (
+  <div
+    className={styles.fill}
+    style={{ background: `rgb(${r}, ${g}, ${b})` }}
+  >
+    <p>{`rgb(${r},${g},${b})`}</p>
+  </div>
+  );
 }
 
 function ScreenColor() {
-  return <div>{/* Create Route Inside Switch */}</div>;
+  return <div>{ <Route path="/rgb/:r/:g/:b">
+    <RGB/>
+  </Route> }</div>;
```

Comment:
you can wrap the `Route` in a `Switch` and get rid of the `{ }` around the `Route`.